test/sync-routes.test.js

```diff
@@ -1,32 +1,32 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const Fastify = require('..')
 
 test('sync route', async t => {
   const app = Fastify()
-  t.teardown(app.close.bind(app))
   app.get('/', () => 'hello world')
   const res = await app.inject('/')
-  t.equal(res.statusCode, 200)
-  t.equal(res.body, 'hello world')
+  t.assert.strictEqual(res.statusCode, 200)
+  t.assert.strictEqual(res.body, 'hello world')
+  app.close()
```

Comment:
we should use `t.after(() => fastify.close())` everywhere: in this PR these syntaxes are switching 😞

Reply:
I updated `sync-routes.test.js` file with `t.after(() => fastify.close())`